themes/aframe/layout/partials/toc.ejs

```diff
@@ -0,0 +1,22 @@
+<% var toc = table_of_contents(page.content); %>
+
+<% if (toc.length) { %>
+  <h2>Table of Contents</h2>
+
+  <ul id="table-of-contents">
```

Comment:
because the `id` is on the `ul` instead of the `h2`, it's not clickable. can we switch that around? and then you can change your [CSS selector](https://github.com/aframevr/aframe-site/pull/276/files#diff-0cc9234c5fe6e28e268a3d89fce00c9fR498) to be `#table-of-contents + ul a`